### ISSUE TITLE
After recent update, `\boldsymbol \partial` renders badly in MathJax 3 (tex-chtml)

### ISSUE THREAD

**Issue by MichaelAllenWarner** [mathjax/MathJax#2417]
### Issue Summary

A few weeks ago, `\boldsymbol \partial` was giving beautiful browser-consistent results in MathJax 3 (tex-chtml), and it looked to be simply the boldface version of the `\partial` symbol, exactly as expected.

Today, I noticed that it's now giving poor and browser-inconsistent results, and looking at the source code I see that the character is rendered not as the `content` of a `::before` pseudo-element (as is the case with other characters in the equations), but rather as a literal `𝛛` character in a `Text` node.

### Steps to Reproduce:

1. Use the `https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js` component.
2. Use `\boldsymbol \partial` in an equation.

### Technical details:

* MathJax Version: `mathjax@3/es5/tex-chtml.js`

### Supporting information:

 * Live link: https://michaelallenwarner.github.io/physics/2019/07/16/vector-triple-products-in-minkowski-spacetime.html#the-double-curl
 * Screenshots of inconsistent browser results (Chrome vs. Firefox):
![Screen Shot 2020-04-21 at 10 01 55 AM](https://user-images.githubusercontent.com/44578578/79878591-98913e80-83bb-11ea-9a66-c6c764caa048.png)
![Screen Shot 2020-04-21 at 10 02 16 AM](https://user-images.githubusercontent.com/44578578/79878592-9929d500-83bb-11ea-917d-362950dcd481.png)

 * MathJax config: https://github.com/MichaelAllenWarner/MichaelAllenWarner.github.io/blob/master/assets/mathjax-prescript.js
 * Script tag: https://github.com/MichaelAllenWarner/MichaelAllenWarner.github.io/blob/master/_includes/head.html#L44

**Comment by dpvc** [comment]
Thanks for the report.  This is due to changes in the font handling in order to allow MathML `mathvariant` attributes to select greek letters better (including upright ones, which are not in the default MathJax fonts).  The `\partial` macro should produce `mathvariant="italic"` but currently does `mathvariant="normal"`, so when `\boldsymbol` emboldens it, it gets `mathvariant="bold"` rather than `mathvariant="bold-italic"` where the proper character can be found.

A work around for now would be to add

``` html
<script>
MathJax = {
  startup: {
    ready() {
      MathJax.startup.defaultReady();
      const font = MathJax.startup.output.font;
      font.getChar('bold', 0x2202)[3].smp = font.getChar('bold-italic', 0x2202)[3].smp;
    }
  }
};
</script>
```

(or merge it into your current configuration).  This will put the bold-italic partial into the bold font for now.

**Comment by MichaelAllenWarner** [comment]
Like a charm—many thanks.

**Comment by dpvc** [comment]
Fixed in v3.1 released today.
